OloEngine/src/OloEngine/Core/Hash.cpp

```
@@ -72,4 +72,27 @@ namespace OloEngine
         return CRC32(string.c_str());
     }
 
+    u32 Hash::CRC32(const void* data, sizet size)
+    {
+        OLO_PROFILE_FUNCTION();
+
+        if (!data)
```

Clang (14.0.6)

[warning] implicit conversion 'const void *' -> bool

(readability-implicit-bool-conversion)